### PR TITLE
Update mysql helm chart to bitnami

### PR DESCRIPTION
**Pull request by ankitjain235**
## Change Overview

This PR updates the MySQL helm chart to be used from bitnami repo. With this PR, all the app/blueprints are moved from stable and incubator charts to their respective repos.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [x] :robot: Test

## Issues

- #XXX

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [x] :green_heart: E2E

- Tested with manual steps as per README, using the bitnami chart.
- Tested with integration test:
` $ make integration-test MySQL`
Output: https://gist.github.com/ankitjain235/7511106aa35bc012d645edf9ffa3eeeb

### CODE REVIEW

File: examples/stable/mysql/README.md [hunk 1]
```diff
@@ -15,38 +15,31 @@ This chart bootstraps a single node MySQL deployment on a [Kubernetes](http://ku
...
 
-By default a random password will be generated for the root user. For setting your own password, use the `mysqlRootPassword` param as shown above.
+By default a random password will be generated for the root user. For setting your own password, use the `root.password` param as shown above.
 
 You can retrieve your root password by running the following command. Make sure to replace [YOUR_RELEASE_NAME] and [YOUR_NAMESPACE]:
 
     `kubectl get secret [YOUR_RELEASE_NAME] --namespace [YOUR_NAMESPACE] -o jsonpath="{.data.mysql-root-password}" | base64 --decode`
 
-> **Tip**: List all releases using `helm list` (Helm Version 2) or `helm list --all-namespaces`, in case of Helm Version 3.
+> **Tip**: List all releases using `helm list --all-namespaces`, in case of Helm Version 3.
```
Comment (PrasadG193) [inline examples/stable/mysql/README.md:42]:
```suggestion
> **Tip**: List all releases using `helm list --all-namespaces`.
```
If we are planning to deprecate helm2 support

Comment (viveksinghggits) [inline examples/stable/mysql/README.md:42]:
```suggestion
> **Tip**: List all releases using `helm list --all-namespaces`, using Helm Version 3.
```

File: examples/stable/mysql/mysql-blueprint.yaml [hunk 2]
```diff
@@ -57,7 +57,7 @@ actions:
         - |
           s3_path="{{ .ArtifactsIn.mysqlCloudDump.KeyValue.s3path }}"
           root_password="{{ index .Phases.restoreFromBlobStore.Secrets.mysqlSecret.Data "mysql-root-password" | toString }}"
-          kando location pull --profile '{{ toJson .Profile }}' --path ${s3_path} - | gunzip | mysql -u root --password=${root_password} -h {{ .Deployment.Name }}
+          kando location pull --profile '{{ toJson .Profile }}' --path ${s3_path} - | gunzip | mysql -u root --password=${root_password} -h {{ index .Object.metadata.labels "release" | toString }}
```
Comment (viveksinghggits) [inline examples/stable/mysql/mysql-blueprint.yaml:60]: do we not have access to mysql service using `StatefulSet.Name`

File: examples/stable/mysql/README.md [hunk 3]
```diff
@@ -128,7 +121,7 @@ s3-profile-drnw9   2m
 
 # Create Actionset
 # Please make sure the value of profile and blueprint matches with the names of profile and blueprint that we have created already
-$ kanctl create actionset --action backup --namespace kanister --blueprint mysql-blueprint --deployment mysql-test/mysql-release --profile mysql-test/s3-profile-drnw9 --secrets mysql=mysql-test/mysql-release
+$ kanctl create actionset --action backup --namespace kasten-io --blueprint mysql-blueprint --statefulset mysql-test/mysql-release-master --profile mysql-test/s3-profile-drnw9 --secrets mysql=mysql-test/mysql-release
```
Comment (viveksinghggits) [inline examples/stable/mysql/README.md:124]: this should be in kanister namespace right. I dont think we are creating kasten-io namespace in this readme.

**Review by viveksinghggits — CHANGES_REQUESTED** [review]

File: examples/stable/mysql/README.md (hunk 3 above)
Comment (ankitjain235) [reply examples/stable/mysql/README.md:124]: Right, at one of the places, `kasten-io` was being used for namespace. Will update these.

File: examples/stable/mysql/mysql-blueprint.yaml (hunk 2 above)
Comment (ankitjain235) [reply examples/stable/mysql/mysql-blueprint.yaml:60]: The service is named after release name, while statefulset is named as "release-name"-master.

**Review by viveksinghggits — APPROVED** [review]

File: examples/stable/mysql/mysql-blueprint.yaml [hunk 4]
```diff
@@ -45,8 +45,13 @@ actions:
...
           namespace: '{{ .StatefulSet.Namespace }}'
       args:
+<<<<<<< HEAD
         image: kanisterio/mysql-sidecar:0.40.0
         namespace: "{{ .StatefulSet.Namespace }}"
+=======
+        image: kanisterio/mysql-sidecar:0.41.0
+        namespace: "{{ .Deployment.Namespace }}"
+>>>>>>> master
         command:
```
Comment (viveksinghggits) [inline examples/stable/mysql/mysql-blueprint.yaml:50]: @ankitjain235 
something wrong here?

Comment (ankitjain235) [reply examples/stable/mysql/mysql-blueprint.yaml:50]: Accidentally missed this merge while rebasing with master. Fixed in the last commit.

**Pull request merged**